packages/slice-machine/.depcheckrc

```diff
@@ -0,0 +1,12 @@
+ignores: [
```

Comment:
that's interesting !
Do you think adding all those ignores is a good sign or a bad sign ?

Could it be that dependencies are misplaced and installed by other packages ?

Reply:
Some of them (newtype-ts, and monocle-ts) were added in this PR by Marc, so I didn't want to re-create that problem: https://github.com/prismicio/slice-machine/pull/620/files

@svgr/webpack is used by the next config. Not sure why depcheck ignores this.

rollup and nodemon are used in the package.json so they aren't recognised by depcheck

I've now removed the types one, because I didn't realise that depcheck actually solves them properly. 

The last 3 (esbuild, babel-plugin-module-resolver and typescript-is) I'm honestly just worried about deleting since they sound important 😂

Reply:
Got help from someone here. The babel-plugin is being used by the the babel config so needs to stay. typescript-is looks like it's probably not since it isn't referenced anywhere in the ts-config.

esbuild looks like it might also not be used since it's not referenced anywhere in the `package.json` build scripts. So I'll try to remove it also

Reply:
hey,

I don't have a lot of knowledge about dependencies because appart from the one we uses for tests, they don't change much.
I believe esbuild might have been used before we switched to rollup.

[Typescript-is](https://www.npmjs.com/package/typescript-is) seems to be for run-time validation of objects
We already do that using [io-ts](https://www.npmjs.com/package/io-ts) so I guess we don't need it if it's not used 😄

Reply:
If you need to test the APP with less dependencies sees if everything works well, you can
- Test in dev mode
- Test in bundled mode
  - For that, run `npm run bundle` in the Slice Machine package
  - Use `npm link` to link slice machine to a side project and just run it to test if it still works 😃

Reply:
Seems good to me I believe